bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        uri="./milvus_demo_metadata.db",
</pre><p>nit, why "metadata", e.g. is this demo specifically about metadata storage or filtering? otherwise it feels a little unrelated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/#comment-1705463983'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Yeah, it was because I had another demo that was running and the db was called <code>milvus_demo.db</code>,I'll change it.   </p>